samples/nrf9160/modem_shell/boards/thingy91_nrf9160_ns.conf

```diff
@@ -13,6 +13,7 @@ CONFIG_MOSH_CURL=n
 CONFIG_MOSH_REST=n
 CONFIG_MOSH_SMS=n
 CONFIG_MOSH_IPERF3=n
+CONFIG_MOSH_GNSS=n
```

Comment:
Because GNSS is disabled, nRF Cloud A-GPS isn't needed either. Disabling that would free a few additional kilobytes. The current state:
```
Memory region         Used Size  Region Size  %age Used
           FLASH:      348700 B     380416 B     91.66%
            SRAM:       95712 B     203544 B     47.02%
        IDT_LIST:          0 GB         2 KB      0.00%
```
With `CONFIG_NRF_CLOUD_AGPS` disabled:
```
Memory region         Used Size  Region Size  %age Used
           FLASH:      341256 B     380416 B     89.71%
            SRAM:       89408 B     203544 B     43.93%
        IDT_LIST:          0 GB         2 KB      0.00%
```

Reply:
Location is enabled which "needs" AGPS

Reply:
You are right, silly me.